maplayer.c

```diff
@@ -1604,9 +1604,19 @@ char* msLayerBuildSQLOrderBy(layerObj *layer)
   if( layer->sortBy.nProperties > 0 ) {
     int i;
     for(i=0;i<layer->sortBy.nProperties;i++) {
-      char* escaped = msSmallMalloc(strlen(layer->sortBy.properties[i].item) + 3);
-      sprintf(escaped, "\"%s\"", layer->sortBy.properties[i].item);
-      if( i > 0 )
+      char* escaped = msLayerEscapePropertyName(layer, layer->sortBy.properties[i].item);
+      //Enclose property name in double quotes (if it isn't yet) to ensure that mixed case property names are supported
+      if (escaped[0] != '"')
```

Comment:
My suggestion would be to just remove completely msOracleSpatialEscapePropertyName, so that the default implementation that does quoting and escaping mechanism is used.